_test_null_safety/test/common/message.dart

```diff
@@ -4,3 +4,6 @@
 
 // Uses `?` to ensure null safety is enabled
 final String? message = 'hello';
+
+// Prints a non-nullable message, used in null assertion tests.
+void printString(String message) => print(message);
```

Comment:
[nit] how about `printNonNullable` so that the name is a closer match to the intent?

Reply:
Sure